index.html

```diff
@@ -0,0 +1,217 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Jery Mey Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="body-white">
+        <nav class="navbar">
+            <div class="logo">
+              <a href="#">My Logo</a>
+            </div>
+            <div class="menu-icon">
+                <img src="images/appbar.png" alt="menu button" />
+            </div>
+            <ul class="nav-links">
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+       <section class="header">
+        <h1 class="title">I’m Jery Mey. <br>
+            Glad to see <br> you!</h1>
+            <p>
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+            <a class="link" href="#">LET'S CONNECT</a>
+        <ul class="social-icons">
+           <li><a href=""><img src="images/social-media-icon/tw.png" alt="twitter"></a></li>
+           <li><a href=""><img src="images/social-media-icon/linkedin.png" alt="linkedin"></a></li>
+           <li><a href=""><img src="images/social-media-icon/medium.png" alt="medium"></a></li>
+           <li><a href=""><img src="images/social-media-icon/github.png" alt="github"></a></li>
+           <li><a href=""><img src="images/social-media-icon/v.png" alt="v"></a></li>
+        </ul>
+
+      </section>
```

Comment:
- [ ] Kindly readjust the padding/margin bottom of your header so it stick to Figma design 🛩️ 

- [ ] Also it could be better if you reduce the `font-size` of the paragraph in **red** to `16px` as mention in the Figma template 👍 

| Your design | Figma design |
| :---: | :---: |
| ![Capture66](https://github.com/jerymey/Portfolio-setup/assets/80458199/35362711-e1b7-497d-940e-c92b48f349bf) | ![Capture5](https://github.com/jerymey/Portfolio-setup/assets/80458199/644a5b5a-04aa-4ae7-ac68-1fd2d45f07ae) |